vidarr-cromwell/README.md

```diff
@@ -31,7 +31,7 @@ using `"workflowSource"`. `"workflowOptions"` is the workflow options Cromwell
 requires. Although Cromwell is given the entire workflow, it still needs to
 know the WDL version, provided as `"wdlVersion"`. The workflow will be called
 with two arguments, the file to copy (which will be provided as the argument
-(`"fileField"`) and the submitter-provided archive directory
+(`"fileField"`)) and the submitter-provided archive directory
```

Comment:
Extra `)`

Reply:
The first ) matches with `(`"file` and the second matches with `(which`. This was flagged by intellij because the `(which` is never closed